runtime-light/stdlib/string/regex-functions.cpp

```diff
@@ -493,6 +491,81 @@ bool replace_regex(RegexInfo& regex_info, uint64_t limit) noexcept {
   return true;
 }
 
+bool add_output_val(RegexInfo& regex_info, bool no_empty, bool offset_capture, std::size_t start_offset, std::size_t end_offset) {
```

Comment:
In general, I think this is a bad design to have a function that implicitly adds something unknown to something unknown. At least, we can say nothing about this function without looking at its implementation